packages/launch-screen/default-behavior.js

```diff
@@ -6,7 +6,7 @@ var handle = LaunchScreen.hold();
 
 var Template = Package.templating && Package.templating.Template;
 
-Meteor.startup(function () {
+Meteor.startup(async function () {
```

Comment:
Is this a safe change? I mean, `Meteor.startup` doesn't handle async functions in any way, i.e., it runs them and forgets about the results. Making all `startup` calls async will potentially change the ordering, which may impact both the behavior and the performance. Also, as it's not waiting for the results, it'll make `Call Meteor.startup hooks` profiling section useless.

Reply:
We will don't make these changes to the client side.
So I will close this PR.

About call async in `Meteor.startup`: 
Yes, the order will change, but we will work on it. First on the server side and then on the client side.

Thank you for your review.